airbyte-integrations/connectors/source-intercom/manifest.yaml

```diff
@@ -661,6 +662,7 @@ definitions:
         cursor_datetime_formats:
           - "%s"
         is_client_side_incremental: true
+        global_substream_cursor: true
```

Comment:
Confirming that this is done cause there are usually so many conversations that we assume we will hit the per partition limit and therefore we're fine with having a global state. 

Benefits:
* Easier to understand state

Drawback:
* Duplication that would not have occurred for people with very small syncs where we don't this the per partition limit